src/main/java/crawlercommons/sitemaps/sax/DelegatorHandler.java

```diff
@@ -170,19 +172,46 @@ else if ("channel".equals(localName)) {
         delegate.setExtensionNamespaces(extensionNamespaces);
     }
 
+    @Override
     public void endElement(String uri, String localName, String qName) throws SAXException {
         if (delegate != null) {
             delegate.endElement(uri, localName, qName);
         }
         elementStack.pop();
     }
 
+    @Override
     public void characters(char ch[], int start, int length) throws SAXException {
         if (delegate != null) {
             delegate.characters(ch, start, length);
         }
     }
 
+    protected void appendCharacterBuffer(char ch[], int start, int length) {
+        if (characterBuffer != null) {
```

Comment:
When would `characterBuffer` ever be null? Same question for `getAndResetCharacterBuffer()`

Reply:
Ok, I'll update the PR.